server/src/main/java/org/apache/iotdb/db/sync/sender/transfer/SyncClient.java

```diff
@@ -339,27 +339,44 @@ public void confirmIdentity() throws SyncConnectionException {
 
   /** UUID marks the identity of sender for receiver. */
   private String getOrCreateUUID(File uuidFile) throws IOException {
-    String uuid;
     if (!uuidFile.getParentFile().exists()) {
       uuidFile.getParentFile().mkdirs();
     }
-    if (!uuidFile.exists()) {
-      try (FileOutputStream out = new FileOutputStream(uuidFile)) {
-        uuid = generateUUID();
-        out.write(uuid.getBytes());
-      } catch (IOException e) {
-        logger.error("Cannot insert UUID to file {}", uuidFile.getPath());
-        throw new IOException(e);
+
+    boolean hasCreatNewUuidFile = false;
+    for (int i = 0; i < 2; i++) {
+      if (!uuidFile.exists()) {
```

Comment:
hi , why use for 2 twice?

Reply:
Original thought is to read uuid from uuidfile again after create new uuidfile.
Please read new submit codes, have changed this part.

Reply:
@wangchao316   How about new code change?   :-)